### DISCUSSION TITLE
nginx reverse prox to subfolder

### DISCUSSION THREAD

**Discussion by ngrossmann99** [jonaswinkler/paperless-ng#740] [Support] [answered]
Hello, I'm trying to setup an nginx reverse proxy to my docker container running paperless. I want to access paperless from a subfolder on my domain. I'm using the swag docker container for the nginx proxy.
Here's my paperless.subfolder.conf:
```
location /paperless {
    return 301 $scheme://$host/paperless/;
}
location ^~ /paperless/ {
    include /config/nginx/proxy.conf;
    resolver 127.0.0.11 valid=30s;
    set $upstream_app paperless;
    set $upstream_port 8000;
    set $upstream_proto http;
    proxy_pass $upstream_proto://$upstream_app:$upstream_port;

}
```
when I try to open mydomain.com/paperless it redirects me to the login page but not on /paperless/accounts but to mydomain.com/accounts. I've already set 
```
PAPERLESS_FORCE_SCRIPT_NAME=/paperless
``` 
and 
```
PAPERLESS_STATIC_URL?/paperless/static/
``` 
and I've tried this
```
if ($http_referer ~* /paperless) {
    rewrite ^/accounts/(.*) /paperless/accounts/$1? break;
}
```
in my nginx config but it always redirects me to the root of my server. I can't figure out what I'm doing wrong, I hope someone can help me with this

**Comment by jonaswinkler** [comment, marked as answer]
You're not doing anything wrong. This is not supported yet, since I'm having some troubles with redirecting certain parts of the front end to sub paths (namely anything in `/assets/`). See also #36.

**Reply by ngrossmann99** [reply]
Ah okay thanks
